packages/core/contracts-ovm/insured-bridge/implementation/BridgePool.sol

```diff
@@ -136,19 +156,7 @@ contract BridgePool is Testable {
         // Save new deposit:
         newDeposit.depositState = DepositState.PendingSlow;
         newDeposit.depositType = DepositType.Slow;
-        newDeposit.relayData = RelayAncillaryDataContents({
-            depositId: depositId,
-            l2Sender: l2Sender,
-            recipient: recipient,
-            depositTimestamp: depositTimestamp,
-            l1Token: l1Token,
-            amount: amount,
-            maxFeePct: maxFeePct,
-            proposerRewardPct: proposerRewardPct,
-            realizedFeePct: realizedFeePct,
-            slowRelayer: msg.sender
-        });
-        bytes memory priceRequestAncillaryData = getRelayAncillaryData(newDeposit.relayData);
+        newDeposit.relayHash = getRelayHash(relayData);
```

Comment:
Why do we need to store this if it's passed in to every method?

Also, it might make things easier if we add this hash to the DepositRelayed event as an indexed parameter so it's easy to find and associate to the other data offchain.

Reply:
Great catch, I removed this param and added the ancillary data to the DepositRelayed event